src/puppetlabs/puppetdb/cli/metrics_service.clj

```diff
@@ -0,0 +1,23 @@
+(ns puppetlabs.puppetdb.cli.metrics-service
+  (:require [clojure.tools.logging :as log]
+            [puppetlabs.puppetdb.http.metrics.server :as server]
+            [puppetlabs.trapperkeeper.core :refer [defservice main]]
+            [compojure.core :as compojure]
+            [puppetlabs.puppetdb.cli.services :refer [build-whitelist-authorizer]]))
+
+(defprotocol MetricsServer)
```

Comment:
If you're only a consumer of other services, no. Here's an example: https://github.com/puppetlabs/puppetdb/blob/stable/src/com/puppetlabs/puppetdb/cli/services.clj#L357